src/assets/translations/oc.json

```diff
@@ -0,0 +1,65 @@
+{
+    "advertising": {
+        "admessage": "La publicitat s’acaba dins xx",
+        "cuetext": "Publicitat",
+        "displayHeading": "Publicitat",
+        "loadingAd": "Cargament de la publicitat",
+        "podmessage": "Publicitat __AD_POD_CURRENT__ sus __AD_POD_LENGTH__.",
+        "skipmessage": "Sautar la publicitat dins xx",
+        "skiptext": "Sautar"
+    },
+    "airplay": "AirPlay",
+    "audioTracks": "Pistas àudio",
+    "auto": "Automatic",
+    "buffer": "Cargament",
+    "cast": "Chromecast",
+    "cc": "Sos-títols",
+    "close": "Tampar",
+    "errors": {
+        "badConnection": "A causa d’un problèma de connexion Internet, la vidèo pòt pas èsser legida.",
+        "cantLoadPlayer": "Desolat, la vidèo a pas pogut se cargar.",
+        "cantPlayInBrowser": "Aqueste navigador es pas pres en carga.",
+        "cantPlayVideo": "La vidèo pòt pas èsser legida.",
+        "errorCode": "Còdi d'error",
+        "liveStreamDown": "Lo streaming en live es en mantenença o es acabat.",
+        "protectedContent": "Avèm pas pogut accedir al contengut protegit.",
+        "technicalError": "Una error tecnica empacha la lectura de la vidèo."
+    },
+    "exitFullscreen": "Sortir del Mòde Ecran complèt",
+    "fullscreen": "Ecran complèt",
+    "hd": "Qualitat",
+    "liveBroadcast": "Dirècte",
+    "logo": "Logo",
+    "mute": "Copar lo Son",
+    "next": "Seguent",
+    "nextUp": "Vidèo venenta",
+    "notLive": "Fòra dirècte",
+    "off": "Dsactivar",
```

Comment:
@Quenty31  should this be Desactivar ? it seems like a letter might be missing after the 'D'

Reply:
Yes, sorry!